scripts/babel/remove-getters.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+module.exports = ({types: t}) => {
+  return {
+    visitor: {
+      ObjectMethod: path => {
```

Comment:
This might match even other things that are supposed to stay getters because it will visit every method on every object, but I don't know what are FB's requirements here.

I don't know what the generated code looks like at this step, but it could be better to collect more information about the node's ancestry before deciding it's the one.

Reply:
Any code help to improve this is much appreciated. I am not sure either for the requirements. 

The code is currently using the `Rollup legacy` option to apply *remove getters*, which is now deprecated.